dev.env.example

```diff
@@ -73,3 +76,7 @@ MATOMO_URL=http://matomo
 
 EXPORT_BUCKET_NAME=crossfeed-local-exports
 IS_LOCAL=1
+
```

Comment:
PE_DB_NAME, PE_DB_USERNAME and PE_DB_PASSWORD are defined twice in the file. Is this a mistake?

Reply:
Yeah, it's a mistake. Can you remove the first definition of them?